packages/fxa-payments-server/server/lib/amplitude.js

```diff
@@ -137,3 +168,9 @@ module.exports = (event, request, data) => {
     statsd.increment('amplitude.event.dropped');
   }
 };
+
+module.exports = {
+  amplitude,
+  getLocation,
+  getCountryCode,
+};
```

Comment:
I changed the exports for this file in order to test the two new helpers, `getLocation` and `getCountryCode`.